### PR TITLE
[stable/minio] add custom labels on ingress

### PR DESCRIPTION
**Pull request by christophemorio**
#### What this PR does / why we need it:

The Ingress labels are basically `app`, `chart`, `release`, `heritage` .

When working with *labelselector*, one needs to append others custom defined labels.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by nitisht — CHANGES_REQUESTED** [review]
Can you please add documentation in readme about how to use this, with an example

**Comment by mathieuherbert** [comment]
@nitisht can you help us on the ci issue please ?

File: stable/minio/templates/ingress.yaml [hunk 1]
```diff
@@ -11,6 +11,9 @@ metadata:
     chart: {{ template "minio.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- with .Values.ingress.customlabels }}
```
Comment (nitisht) [inline stable/minio/templates/ingress.yaml:14]: shouldn't this be `{{- if .Values.ingress.customlabels }}` ?

Comment (christophemorio) [reply stable/minio/templates/ingress.yaml:14]: these both are valid, I can opt for `if` syntax if more relevant

```
{{- with .Values.ingress.customlabels }}
{{ toYaml . | indent 4 }}
{{- end }}
```

```
{{- if .Values.ingress.customlabels }}
{{ toYaml .Values.ingress.customlabels | indent 4 }}
{{- end }}
```

**Pull request closed without merging**